### DISCUSSION TITLE
firebase cloud messaging notifications sdk <30

### DISCUSSION THREAD

**Discussion by ahmedbarwari** [Sketchware-Pro/Sketchware-Pro#729] [Q&A]
can someone please let me know how to fix firebase cloud messaging in adk 30 and above? notifications working only with sdk 29 and below. please!

**Comment by DevVigilante** [comment]
> can someone please let me know how to fix firebase cloud messaging in adk 30 and above? notifications working only with sdk 29 and below. please!

I think this is due to Android 10 background app restriction
